src/main/java/org/isf/medicalinventory/gui/InventoryEdit.java

```diff
@@ -865,8 +874,44 @@ private JButton getCloseButton() {
 		});
 		return closeButton;
 	}
-	
-	private JButton getCleanTableButton() {
+
+	private JButton getPrintButton() {
+		printButton = new JButton(MessageBundle.getMessage("angal.common.print.btn"));
+		printButton.setMnemonic(MessageBundle.getMnemonic("angal.common.print.btn.key"));
+		printButton.setEnabled(true);
+
+		printButton.addActionListener(e -> {
+			if (inventory == null || inventory.getId() <= 0) {
+				MessageDialog.info(this, "angal.inventory.pleasesavebeforprinting");
+				return;
+			}
+
+			if (InventoryStatus.done.name().equals(inventory.getStatus())) {
```

Comment:
Why to print only when DONE status? This make non sense, maybe you meant the opposite?

```suggestion
			if (!InventoryStatus.done.name().equals(inventory.getStatus())) {
```